dead_mans_switch_client/__manifest__.py

```diff
@@ -1,13 +1,14 @@
 # -*- coding: utf-8 -*-
 # © 2015-2016 Therp BV <http://therp.nl>
+# © 2017 Avoin.Systems - Miku Laitinen
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 {
     "name": "Dead man's switch (client)",
-    "version": "9.0.1.0.1",
-    "author": "Therp BV,Odoo Community Association (OCA)",
+    "version": "10.0.1.0.0",
+    "author": "Therp BV, Avoin.Systems, Odoo Community Association (OCA)",
```

Comment:
I do appreciate the port to version 10, and I think adding yourself as a contributor is fine, but personally I would not put myself in the author's list for a change like this.

Reply:
I agree with you. I don't contribute that often to OCA so I'm not always up to the guidelines.

I will remove our company from the authors. Is it ok if I keep the mention in other files?